src/tests/api/test_subevents.py

```diff
@@ -50,6 +109,242 @@ def test_subevent_list(token_client, organizer, event, subevent):
+
+    resp = token_client.patch(
+        '/api/v1/organizers/{}/events/{}/subevents/{}/'.format(organizer.slug, event.slug, subevent.pk),
+        {
+            "presale_start": "2017-12-27T10:00:00Z",
+            "presale_end": "2017-12-26T10:00:00Z"
+        },
+        format='json'
+    )
+    assert resp.status_code == 400
+    assert resp.content.decode() == '{"non_field_errors":["The event\'s presale cannot end before it starts."]}'
+
+    resp = token_client.patch(
+        '/api/v1/organizers/{}/events/{}/subevents/{}/'.format(organizer.slug, event.slug, subevent.pk),
+        {
+            "meta_data": {
+                meta_prop.name: "Conference"
+            }
+        },
+        format='json'
+    )
+    assert resp.status_code == 200
+    assert organizer.events.get(slug=event.slug).subevents.get(id=resp.data['id']).meta_values.filter(
+        property__name=meta_prop.name, value="Conference"
+    ).exists()
+
+    resp = token_client.patch(
+        '/api/v1/organizers/{}/events/{}/subevents/{}/'.format(organizer.slug, event.slug, subevent.pk),
+        {
+            "meta_data": {
+            }
+        },
+        format='json'
+    )
+    assert resp.status_code == 200
+    assert not organizer.events.get(slug=event.slug).subevents.get(id=resp.data['id']).meta_values.filter(
+        property__name=meta_prop.name
+    ).exists()
+
+    resp = token_client.patch(
+        '/api/v1/organizers/{}/events/{}/subevents/{}/'.format(organizer.slug, event.slug, subevent.pk),
+        {
+            "meta_data": {
+                "test": "test"
+            }
+        },
+        format='json'
+    )
+    assert resp.status_code == 400
+    assert resp.content.decode() == '{"meta_data":["Meta data property \'test\' does not exist."]}'
+
+    resp = token_client.patch(
+        '/api/v1/organizers/{}/events/{}/subevents/{}/'.format(organizer.slug, event.slug, subevent.pk),
+        {
+            "item_price_overrides": [
+                {
+                    "item": item.pk,
+                    "price": "99.99"
+                }
+            ],
+        },
+        format='json'
+    )
+    assert resp.status_code == 200
+    assert organizer.events.get(slug="dummy").subevents.get(id=resp.data['id']).items.get(id=item.pk).default_price == Decimal('23.00')
+    assert organizer.events.get(slug="dummy").subevents.get(id=resp.data['id']).item_price_overrides[item.pk] == Decimal('99.99')
+
+    resp = token_client.patch(
+        '/api/v1/organizers/{}/events/{}/subevents/{}/'.format(organizer.slug, event.slug, subevent.pk),
+        {
+            "item_price_overrides": [
+                {
+                    "item": 123,
+                    "price": "99.99"
+                }
+            ],
+        },
+        format='json'
+    )
+    assert resp.status_code == 400
+    assert resp.content.decode() == '{"item_price_overrides":[{"item":["Invalid pk \\"123\\" - object does not exist."]}]}'
```

Comment:
Could you also add a test for an item that doesn't belong to the event? you can look in ``test_items.py``, there are fixtures ``event2`` and ``item2`` you can copy over for exactly that purpose.